src/components/Confirm/index.js

```diff
@@ -21,12 +21,17 @@ const FileViewer = ({file:{preview, type}}) =>
     </embed>
   </object>
 
-
-const CaptureViewerPure = ({capture:{blob}}) =>
+const CaptureViewerPure = ({capture:{blob, base64}}) =>
```

Comment:
Matter of preference tbh, I prefer to separate any state code from presentation code.

I think the parameters can be simplified.